stories/core_components/GeoChart.tsx

```diff
@@ -120,7 +120,7 @@ function getPageWithExecution(execution: Execution.IExecutionResponses) {
 function renderChart(config: IGeoConfig, execution: Execution.IExecutionResponses): React.ReactElement {
     return screenshotWrap(
         <div style={wrapperStyle}>
-            <GeoChart
+            <GeoChartInner
```

Comment:
Please rename below 
from `storiesOf("Core components/GeoChart", module)`
to `storiesOf("Core components/GeoChartInner", module)`
in this file